v0.0.1/model-schema.json

```diff
@@ -0,0 +1,192 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/main/v0.0.1/model-schema.json",
+  "title": "ForecastHub model metadata",
+  "description": "This is the schema of the model metadata file, please refer to https://github.com/epiforecasts/covid19-forecast-hub-europe/wiki/Metadata[an] for more information.",
+  "type": "object",
+  "properties": {
+    "schema_version": {
+      "type": "string",
+      "format": "uri"
+    },
+    "team_name": {
+      "description": "The name of the team submitting the model",
+      "type": "string"
+    },
+    "team_abbr": {
+      "description": "Abbreviated name of the team submitting the model",
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9_+]+$",
+      "maxLength": 16
+    },
+    "model_name": {
+      "description": "The name of the model",
+      "type": "string"
+    },
+    "model_abbr": {
+      "description": "Abbreviated name of the model",
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9_+]+$",
+      "maxLength": 16
+    },
+    "model_version": {
+      "description": "Identifier of the version of the model",
+      "type": "string"
+    },
+    "model_contributors": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "affiliation": {
+            "type": "string"
+          },
+          "orcid": {
+            "type": "string",
+            "pattern": "^\\d{4}\\-\\d{4}\\-\\d{4}\\-[\\dX]{4}$"
+          },
+          "email": {
+            "type": "string",
+            "format": "email"
+          },
+          "twitter": {
+            "type": "string"
+          },
+          "additionalProperties": false
+        }
+      }
+    },
+    "website_url": {
+      "description": "Public facing website for the model",
+      "type": "string",
+      "format": "uri"
+    },
+    "repo_url": {
+      "description": "Repository containing code for the model",
+      "type": "string",
+      "format": "uri"
+    },
+    "license": {
+      "description": "License for use of model output data",
+      "type": "string",
+      "enum": [
```

Comment:
Noting that we've had quite a longer list in other projects. e.g. https://github.com/reichlab/covid19-forecast-hub/blob/master/code/validation/accepted-licenses.csv

Reply:
There's a recurring discussion point about which licenses we should accept. We need to restrict the licenses to ones that allow derivative works (e.g. ensemble forecasts).